chunk_forward_tsn.go

```diff
@@ -133,7 +135,7 @@ func (s *chunkForwardTSNStream) unmarshal(raw []byte) error {
 	return nil
 }
 
-func (s *chunkForwardTSNStream) marshal() ([]byte, error) {
+func (s *chunkForwardTSNStream) marshal() ([]byte, error) { // nolint:unparam
```

Comment:
Keeping param to keep style to unmarshal(). don't know if the error param would be ever be used?